src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -80,9 +80,11 @@ function ($deferred, $namespace) {
     /**
      * Deletes all items in the pool.
      *
+     * @param string The prefix used for all identifiers managed by this pool.
+     *
      * @return bool True if the pool was successfully cleared, false otherwise.
      */
-    abstract protected function doClear();
+    abstract protected function doClear($namespace);
```

Comment:
I propose adding $namespace to this abstract method so that adapters can flush only a subpart of their global pool if that applies